components/organisms/ReportAProblem.js

```diff
@@ -73,9 +77,44 @@ export function ReportAProblem(props) {
             value={i18n.language}
           />
           <fieldset>
-            <legend className="text-base sm:text-p font-body font-normal mb-6">
+            <legend className="text-base sm:text-p font-body font-normal">
               {t("reportAProblemCheckAllThatApply", { lng: props.language })}
             </legend>
+            <ul className="list-outside list-disc px-6 py-2">
+              <li className="text-xxs sm:text-sm font-body my-4 leading-tight sm:leading-6">
+                <b>{t("reportAProblemNoReply", { lng: props.language })}</b>
```

Comment:
- Let's add a space {" "} after the `<b>` so that we have a space between the period and the next word ("reply.For")

<img width="371" alt="Screen Shot 2021-07-22 at 16 04 09" src="https://user-images.githubusercontent.com/2454380/126702501-431ecdf7-ccfa-4512-8e51-5113caa4e643.png">